### PR TITLE
sale_delivery_date: deliveryslip report - Add div name

### PR DESCRIPTION
**Pull request by mmequignon**
To ease report override

### CODE REVIEW

File: sale_delivery_date/reports/stock_picking.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <odoo>
     <template id="report_delivery_document" inherit_id="stock.report_delivery_document">
         <xpath expr="//div[@name='div_sched_date']" position="after">
-            <div class="col-auto mw-100 mb-2">
+            <div class="col-auto mw-100 mb-2" name="div_expected_delivery_date">
```
Comment (TDu) [inline sale_delivery_date/reports/stock_picking.xml:7]:
```suggestion
            <div class="col-auto mw-100 mb-2" name="expected_delivery_date">
```

**Review by TDu — APPROVED** [review]
suggestion on naming

**Review by carmenbianca — APPROVED** [review]
LGTM, seconding TDu's suggestion

**Pull request merged**